Source/WebCore/page/PerformanceLogging.cpp

```diff
@@ -39,39 +39,39 @@
 namespace WebCore {
 
 #if !RELEASE_LOG_DISABLED
-static const char* toString(PerformanceLogging::PointOfInterest poi)
+static ASCIILiteral toString(PerformanceLogging::PointOfInterest poi)
 {
     switch (poi) {
     case PerformanceLogging::MainFrameLoadStarted:
-        return "MainFrameLoadStarted";
+        return "MainFrameLoadStarted"_s;
     case PerformanceLogging::MainFrameLoadCompleted:
-        return "MainFrameLoadCompleted";
+        return "MainFrameLoadCompleted"_s;
     }
     RELEASE_ASSERT_NOT_REACHED();
-    return "";
+    return ""_s;
 }
 #endif
 
-HashMap<const char*, size_t> PerformanceLogging::memoryUsageStatistics(ShouldIncludeExpensiveComputations includeExpensive)
+HashMap<ASCIILiteral, size_t> PerformanceLogging::memoryUsageStatistics(ShouldIncludeExpensiveComputations includeExpensive)
 {
-    HashMap<const char*, size_t> stats;
+    HashMap<ASCIILiteral, size_t> stats;
```

Comment:
Not super familiar with this code. I just looked into it and it doesn't appear we need the check for uniqueness. I looked at the 2 call sites: 1 doesn't care about ordering because it merely populates a dictionary with the stats. The other one though does logging and thus ends up printing in random order. I think we can switch to a vector here.